source/includes/mongodb-compatibility-table-php.rst

```diff
@@ -15,7 +15,20 @@
      - MongoDB 3.0
      - MongoDB 2.6
 
+   * - ext 1.12 + lib 1.11
+     - |checkmark|
+     - |checkmark|
+     - |checkmark|
+     - |checkmark|
+     - |checkmark|
+     - |checkmark|
```

Comment:
FYI: 5.1-compat releases for drivers will also drop support for pre-3.6 server versions ([DRIVERS-1315](https://jira.mongodb.org/browse/DRIVERS-1315)).